src/com/lftechnology/java/training/sanish/application/controller/EmployeeController.java

```diff
@@ -0,0 +1,384 @@
+package com.lftechnology.java.training.sanish.application.controller;
+
+import com.lftechnology.java.training.sanish.application.component.Constants;
+import com.lftechnology.java.training.sanish.application.component.EmployeeHelper;
+import com.lftechnology.java.training.sanish.application.dbconnection.DbConnect;
+import com.lftechnology.java.training.sanish.application.component.UserLogin;
+import com.lftechnology.java.training.sanish.application.model.dao.UserDao;
+import com.lftechnology.java.training.sanish.application.model.domain.Employee;
+import com.lftechnology.java.training.sanish.application.model.domain.User;
+import com.lftechnology.java.training.sanish.application.model.domain.UserEmployee;
+import com.lftechnology.java.training.sanish.application.utility.UserInput;
+import com.lftechnology.java.training.sanish.application.view.employee.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Employee controller
+ *
+ * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+ */
+public class EmployeeController {
+    private static final Logger LOGGER = Logger.getLogger(EmployeeController.class.getName());
+
+    private EmployeeController() {
+    }
+
+    /**
+     * Employee dashboard page
+     *
+     * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+     */
+    public static void employeeDashboardPage() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "employeeDashboardPage" is 12 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)